### PR TITLE
feat: Add description on notifications 💬

### PR DESCRIPTION
**Pull request by kosssi**
I add header part on `TogglePane.jsx` to add a description on notifications.

You can see the result:

capture d ecran 2018-08-16 a 16 27 01

### CODE REVIEW

File: src/ducks/settings/TogglePane.styl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.TogglePane__main
+    margin-bottom 3rem
+
+.TogglePane__description
+    color coolGrey
```
Comment (ptbrowne) [inline src/ducks/settings/TogglePane.styl]: We should have a description label in cozy ui for that.

File: src/ducks/settings/TogglePane.styl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.TogglePane__main
+    margin-bottom 3rem
```
Comment (ptbrowne) [inline src/ducks/settings/TogglePane.styl]: Why do we need another class for this ? Can we use the spacing styles from cozy-ui ?

File: src/ducks/settings/TogglePane.jsx [hunk 3]
```diff
@@ -26,12 +27,22 @@ class TogglePane extends Component {
...
   render() {
-    const { rows, settingsCollection, settingsKey, t, title } = this.props
+    const {
+      rows,
+      settingsCollection,
+      settingsKey,
+      t,
+      title,
+      description
+    } = this.props
```
Comment (ptbrowne) [inline src/ducks/settings/TogglePane.jsx]: This is starting to get ugly, I think we should be able to refactor to have less properties here.

File: src/ducks/settings/TogglePane.styl (hunk 2 above)
Comment (ptbrowne) [inline src/ducks/settings/TogglePane.styl]: I don't understand why we need such a space. Maybe it  should be the job of an utility class to do that ? The spacing classes from Cozy-UI do not do the job ?

File: src/ducks/settings/TogglePane.styl (hunk 1 above)
Comment (kosssi) [reply src/ducks/settings/TogglePane.styl]: have you a link to the documentation ?

**Pull request merged**